### PR TITLE
Sort entities, add time and headers in wear settings

### PR DESCRIPTION
**Pull request by dshokouhi**
## Summary

- Add the time and scroll indicator to settings
- Set proper list headers for remaining sections
- Sort entities on home screen

## Screenshots

image

image

## Link to pull request in Documentation repository

Documentation: home-assistant/companion.home-assistant#

### CODE REVIEW

File: wear/src/main/java/io/homeassistant/companion/android/home/views/MainView.kt [hunk 1]
```diff
@@ -64,11 +64,11 @@ fun MainView(
...
-    val scenes = entities.filter { it.key.split(".")[0] == "scene" }.values.toList()
-    val scripts = entities.filter { it.key.split(".")[0] == "script" }.values.toList()
-    val lights = entities.filter { it.key.split(".")[0] == "light" }.values.toList()
-    val inputBooleans = entities.filter { it.key.split(".")[0] == "input_boolean" }.values.toList()
-    val switches = entities.filter { it.key.split(".")[0] == "switch" }.values.toList()
+    val scenes = entities.filter { it.key.split(".")[0] == "scene" }.values.toList().sortedBy { it.entityId }
+    val scripts = entities.filter { it.key.split(".")[0] == "script" }.values.toList().sortedBy { it.entityId }
+    val lights = entities.filter { it.key.split(".")[0] == "light" }.values.toList().sortedBy { it.entityId }
+    val inputBooleans = entities.filter { it.key.split(".")[0] == "input_boolean" }.values.toList().sortedBy { it.entityId }
+    val switches = entities.filter { it.key.split(".")[0] == "switch" }.values.toList().sortedBy { it.entityId }
```
Comment (JBassett) [inline wear/src/main/java/io/homeassistant/companion/android/home/views/MainView.kt:71]: We make a temporary list of entities, sort it, then filter the sorted list.

```suggestion
    val entitieList = entities.values.toList().sortedBy { it.entityId }
    val scenes = entitieList.filter { it.key.split(".")[0] == "scene" }
    val scripts = entitieList.filter { it.key.split(".")[0] == "script" }
    val lights = entitieList.filter { it.key.split(".")[0] == "light" }
    val inputBooleans = entitieList.filter { it.key.split(".")[0] == "input_boolean" }
    val switches = entitieList.filter { it.key.split(".")[0] == "switch" }
```

**Review by JBassett — CHANGES_REQUESTED** [review]

File: wear/src/main/java/io/homeassistant/companion/android/home/views/MainView.kt (hunk 1 above)
Comment (dshokouhi) [reply wear/src/main/java/io/homeassistant/companion/android/home/views/MainView.kt:71]: you were close :) had to replace `key` with `entityId` as we were already using the values to sort by

Comment (JBassett) [reply wear/src/main/java/io/homeassistant/companion/android/home/views/MainView.kt:71]: It was from my phone too, cut me some slack, haha

Comment (dshokouhi) [reply wear/src/main/java/io/homeassistant/companion/android/home/views/MainView.kt]: hahahha for sure :P

**Review by JBassett — APPROVED** [review]

**Pull request merged**